tests/e2e/helpers/api/api-keys.ts

```diff
@@ -0,0 +1,128 @@
+import { t } from 'testcafe';
+import * as request from 'supertest';
+import { AddNewDatabaseParameters } from '../../pageObjects/add-redis-database-page';
+import { Common } from '../../helpers/common';
+import {
+    HashKeyParameters,
+    ListKeyParameters,
+    SetKeyParameters,
+    SortedSetKeyParameters,
+    StreamKeyParameters
+} from '../../pageObjects/browser-page';
+import { getDatabaseByName } from './api-database';
+
+const common = new Common();
+const endpoint = common.getEndpoint();
+
+/**
+ * Add Hash key
+ * @param keyParameters The key parameters
+ * @param databaseParameters The database parameters
+ */
+export async function addHashKeyApi(keyParameters: HashKeyParameters, databaseParameters: AddNewDatabaseParameters): Promise<void> {
+    const databaseId = await getDatabaseByName(databaseParameters.databaseName);
+    const response = await request(endpoint).post(`/instance/${databaseId}/hash`)
+        .send({
+            'keyName': keyParameters.keyName,
+            'fields': keyParameters.fields
+        })
+        .set('Accept', 'application/json');
+
+    await t.expect(response.status).eql(201, 'The creation of new Hash key request failed');
+}
+
+/**
+ * Add Stream key
+ * @param keyParameters The key parameters
+ * @param databaseParameters The database parameters
+ */
+export async function addStreamKeyApi(keyParameters: StreamKeyParameters, databaseParameters: AddNewDatabaseParameters): Promise<void> {
+    const databaseId = await getDatabaseByName(databaseParameters.databaseName);
+    const response = await request(endpoint).post(`/instance/${databaseId}/streams`)
```

Comment:
You can create one generic function for all the below functions.
See that `/instance/${databaseId}/` is common, and the last part is changed.
WDYT?

Reply:
they are related to different types, so better to leave as it is